package.json

```diff
@@ -229,7 +229,7 @@
     "react-router-dom": "^4.3.1"
   },
   "optionalDependencies": {
-    "@folio/plugin-find-contact": "^1.1.0",
-    "@folio/plugin-find-interface": "^1.0.0"
+    "@folio/plugin-find-contact": "^2.0.0",
```

Comment:
maybe it's a good idea to put `*` for optional deps

Reply:
for now depths are ok